web/templates/settings/profile.html

```diff
@@ -37,16 +37,20 @@ <h3 class="ui header">{{ .T "General" }}</h3>
                 required
                 disabled />
             </div>
-            <div class="field">
-              <label>
-                {{ .T "Alternative username (cannot be used for login)" }}
-              </label>
-              <input
-                type="text"
-                name="username_aka"
-                value="{{ $d.username_aka }}"
-                placeholder="{{ .T "Also known as..." }}" />
-            </div>
+            {{ if has .Context.User.Privileges 4 }}
+              <div class="field">
+                <label>
+                  {{ .T "Leaderboard size (In-Game)" }}
+                </label>
+                <input
+                  type="number"
+                  name="leaderboard_size"
+                  id="lbSizeInput"
+                  value="{{ $d.leaderboard_size }}"
+                  oninput="validateLeaderboardSizeInput()"
+                  placeholder="1000" />
```

Comment:
probably makes more sense to make the placeholder value the default maybe? or maybe have it say 1-1000 or something indicative

Reply:
yeah i agree

Reply:
resolved in 84ce8750a41425e0425af2726b9beb152bbe85a4